app/models/case_log.rb

```diff
@@ -241,18 +241,20 @@ def set_derived_fields!
       self.year = startdate.year
     end
     case net_income_known
-    when "Yes – the household has a weekly income"
+    when "Weekly"
       self.incfreq = "Weekly"
       self.incref = nil
-    when "Yes – the household has a monthly income"
+    when "Monthly"
       self.incfreq = "Monthly"
       self.incref = nil
-    when "Yes – the household has a yearly income"
+    when "Annually"
       self.incfreq = "Yearly"
       self.incref = nil
     when "Tenant prefers not to say"
       self.incref = 1
       self.incfreq = nil
+    when "Don’t know"
+      self.incfreq = nil
```

Comment:
Not a bid deal but we could factor out the `self.incfreq = nil`

Reply:
Wouldn't we need to set it?

Reply:
We're only setting it to nil in 2 places, not sure how we'd refactor it to be clearer 🤔

Reply:
I count 5 so let's catch up 😄

Reply:
clarified